### ISSUE TITLE
Downloaded PDF's not respecting margins?

### ISSUE THREAD

**Issue by sunlightspectre** [sumatrapdfreader/sumatrapdf#3582]
See title; I'm pretty close to tearing my hair out, here.   

I work with a instructional documentation; I need a format that supports strikethrough text.
Therefore, epub/mobi are out. PDF works, but...

Whenever I try to open downloaded content that displays correctly in-browser - 

The margins seem to irregularly respect `<br>; `
I've no idea why. Neither UI margin setting in the text seems to do anything, either.
Please let me know if there is a way to fix this.

Let me actually edit this to make it clear:

- If there's a way to have sumatra read strikethrough in epub, awesome, that would fix things nicely.
- If there isn't, is there a way to make sumatra respect the margins of the original document.

Thanks!

**Comment by GitHubRulesOK** [comment]
There are NOT many fonts with struck through / underlined letters, ( ̶H̶e̶l̶l̶o̶ ̶W̶o̶r̶l̶d̶!̶). Thus the strike out or under needs to be a drawn as a vector line over the top of ordinary characters. (not usual in an ePub)

PDF does have ability to draw lines under or over text, but not have margins, (they are simple aVoids)  so converting to PDF may produce jagged edges if justification is not used in the editor/writer, which then causes reader problems due to imbalanced spaces between letters.

**Comment by sunlightspectre** [comment]
For the record, I kludged together a workaround using koodos reader as a portable install _solely_ to read .epubs, which - for whatever reason  - vectorised their text correctly, _somehow!_ As if running off of some sort of ghost-logic, instead of your comprehensive, cute, and easy to understand explanation (seriously! you covered every possible thing I might want to know) answer. Given the only font in question is bog-standard arial...

I don't really know why, but it'll work for the office, it's good as done. 

In the future, I'll specify justifications and see how that goes; for now, thank you again for the comprehensive answer; I realise in retrospect some people see cute as a negative, I should say - the amount of screencaps (of the conversation!) and information provided really warmed my heart. _Comprehensive;_ I learned a lot. I always like that.

Closed, and if anyone else has this very _specific_ problem, and you're dealing with a mass amount of text, from different documents with different sources, many of which have different justifications if any - 

Any portable reader that can set custom margins is a temporary, easy fix while and if you are editing said documents.
